src/main/java/org/quiltmc/loader/impl/QuiltLoaderImpl.java

```diff
@@ -148,6 +149,12 @@ public File getGameDirectory() {
 	 */
 	@Override
 	public Path getConfigDir() {
+		if (configDir == null) {
+			// May be null during tests
+			// If this is in production then things are about to go very wrong.
```

Comment:
I'd rather not require a system property to run tests - since every time someone makes a JUnit test they they would need to add said system property in order to make quilt not crash.

Technically JUnit tests are a bit of an odd case though - since they don't go through the normal launch process (set the game provider, load mods, setup mixins, etc) I'm not sure what a better solution would be?